daemon/daemon_unix.go

```diff
@@ -133,118 +133,137 @@ func (daemon *Daemon) adaptContainerSettings(hostConfig *runconfig.HostConfig, a
 	}
 }
 
-// verifyPlatformContainerSettings performs platform-specific validation of the
-// hostconfig and config structures.
-func verifyPlatformContainerSettings(daemon *Daemon, hostConfig *runconfig.HostConfig, config *runconfig.Config) ([]string, error) {
+func verifyContainerResources(resources *runconfig.Resources) ([]string, error) {
```

Comment:
nvm, sorry for the noise I've seen you left `verifyPlatformContainerSettings` :)